experiments/aws_az/aws_az_chaos/experiment/aws_az_chaos.py

```diff
@@ -0,0 +1,132 @@
+import pkg.types.types  as types
+import pkg.aws_az.types.types as experimentDetails
+import pkg.aws_az.environment.environment as experimentEnv
+import pkg.events.events as events
+import logging
+import pkg.status.application as application
+import chaosLib.litmus.aws_az_chaos.lib.aws_az_chaos as litmusLIB
+import pkg.result.chaosresult as chaosResults
+from pkg.utils import client
+import pkg.utils.common.common as common
+import pkg.aws_status.status as awsStatus
+
+# Experiment contains steps to inject chaos
+def Experiment(clients):
```

Comment:
We should make all the function signatures type-safe by adding the data type of the parameters as well as the return type (if applicable). It will make debugging easier in long run and will also be useful for IntelliSense in VSCode and other IDEs.